config/locales/en.yml

```diff
@@ -1187,69 +1187,183 @@ en:
+                If you don’t have a bank account, it could take up to five months to receive your EIP check by mail.
+                To get your payment quickly via direct deposit,
+                <a
+                  href="https://joinbankon.org/home/remotebanking/"
+                  rel="nofollow" target="_blank" data-track-click="stimulus-irs-online-banking"
+                  data-track-attribute-position="stimulus-how-faq6">
+                  sign up for a bank account online</a>
+                and add your account information on the
+                <a
+                  href="https://www.irs.gov/coronavirus/get-my-payment"
+                  rel="nofollow"
+                  target="_blank"
+                  data-track-click="stimulus-irs-get-my-payment"
+                  data-track-attribute-position="stimulus-how-faq6">
+                  IRS Get My Payment website</a>.
+                If you don’t want to sign up for a bank account, you can link your prepaid debit card or
+                <a
+                  href="https://cash.app/help/us/en-us/1018-cash-app-the-stimulus-and-you"
+                  rel="nofollow"
+                  target="_blank"
+                  data-track-click="stimulus-irs-cash-app"
+                  data-track-attribute-position="stimulus-how-faq6">
+                Cash App</a>
+                instead. Many of our VITA partners can also enroll you in a prepaid debit card when you file your
+                return through GetYourRefund.
+            - title: What if I moved since filing my taxes?
+              content_html: >-
+                If you want your EIP check mailed to you, make sure your address is up to date on the
+                <a
+                  href="https://www.irs.gov/faqs/irs-procedures/address-changes"
+                  rel="nofollow"
+                  target="_blank"
+                  data-track-click="stimulus-irs-address-changes"
+                  data-track-attribute-position="stimulus-faq7">
+                  IRS address change website</a>.
+                There may be delays – it could take up to five months to receive your check by mail.
+
+        header: Get your Stimulus Check (EIP)
         intro:
-          common_questions: We know there are many questions around the Economic Impact Payments (EIP) and we have some answers! We’ve broken down some common questions to help you out.
-          description: This year, many people will receive an Economic Impact Payment (often called a stimulus payment) from the federal government. You’ll get up to $1,200 for yourself ($2,400 for a married couple filing jointly) and you’ll get an additional $500 for every dependent child under the age of 17.
+          common_questions: >-
+            We know there are many questions around the Economic Impact Payments
+            (EIP) and we have some answers! We’ve broken down some common questions
+            to help you out.
+          description:
+            - >-
```

Comment:
This is cool YAML syntax. I'm not sure I understand it, except I clearly understand it well enough since it seems to work.